devenv/datasources.yaml

```
@@ -315,6 +315,13 @@ datasources:
       defaultRegion: eu-west-2
       customMetricsNamespaces: "CWAgent"
 
+  - name: JSON API - Journey Times
+    type: simpod-json-datasource
+    editable: true
+    url: http://0.0.0.0:8000/grafana/journey_times
+    uid: hi1
```

Might be a good idea to use a descriptive name for the datasource uids, given we have the control over it
```suggestion
    uid: json-api-journey-times
```
is the naming convention i've been using

---

yeah good shout!